### ISSUE TITLE
Material time picker is not focusable with a keyboard

### ISSUE THREAD

**Issue by gmoraja** [flutter/flutter#49810]
@goderbauer
@gspencergoog
@lorenzOliveto

**Comment by TahaTesser** [comment]
@darrenaustin  @gmoraja 
This looks to be focusing fine to me, except the dialer, Is the main issue here? 

https://user-images.githubusercontent.com/48603081/144241639-b30555e1-8db6-4348-bf59-60b0cd358d07.mov

**Comment by gmoraja** [comment]
This seems to be working fine now, thank you. I'll let you know if we experience this issue again.

**Comment by TahaTesser** [comment]
Thanks, I feel safe to close the issue
